src/utils/dtsi/dtsiPersonUtils.ts

```diff
@@ -44,3 +44,15 @@ export const dtsiPersonPoliticalAffiliationCategoryDisplayName = (
       return 'Independent'
   }
 }
+
+export const hideStanceSlugs: string[] = []
+
+export function isPoliticianStanceHidden(dtsiSlug: string) {
+  const exemptPolitician = hideStanceSlugs?.find(
+    slug => slug.toLowerCase() === dtsiSlug.toLowerCase(),
+  )
+
+  const isStanceHidden = exemptPolitician !== undefined
+
+  return isStanceHidden
+}
```

Comment:
nit: I'd recommend using a Set here:

```suggestion

const hideStanceSlugs = new Set<string>([])

export function isPoliticianStanceHidden(dtsiSlug: string) {
  return hideStanceSlugs.has(dtsiSlug.toLowerCase())
}
```